docs/design/908-proposal-process.md

```diff
@@ -0,0 +1,410 @@
+```
+
+### Proposal review
+
+Every week, a subset of Alloy [governance] members review open and pending
+proposals as part of the proposal review process. The proposal review process
+may be performed asynchronously or synchronously, up to the discretion of the
+governance members.
+
+The goal of proposal review is to make sure proposals are moving forward and
+are receiving attention from the right people. Governance members participating
+in proposal review should cc relevant maintainers, raise important questions,
+ping on stale discussions, and generally drive discussion towards some kind of
+consensus.
+
+The proposal review process also identifies when consensus has been reached on
+a proposal and moves that proposal to the next stage.
+
+The Alloy governance team can, at their discretion, make exceptions that do not
+need to go through all stages, fast-tracking them to any other stage. This may
+be appropriate for proposals that do not merit the full review process or
+proposals that need to be considered quickly, such as an upcoming release
+deadline.
+
+Proposals are in one of the following states:
+
+* **Incoming**: New proposals are given the _Incoming_ state.
+
+  The proposal review process prioritizes proposals in other states to keep
+  things moving forward. If time permits during the review process, proposals
+  in _Incoming_ are moved to _Active_.
+
+* **Active**: Proposals in the _Active_ state are reviewed during the proposal
+  review process to watch for emerging consensus in the discussion. The
+  proposal review group may also comment, make suggestions, ask clarifying
+  questions, and try to restate the proposal to make sure everyone agrees about
+  what exactly is being discussed.
+
+* **Likely Accept**: If a proposal discussion seems to have reached a consensus
+  to accept a proposal, the proposal review group moves the proposal to the
+  _Likely Accept_ state. This change should be indicated with a comment on the
+  proposal. This state marks the final period for comments that might change
+  the recognition of consensus.
+
+* **Likely Decline**: If a proposal discussion seems to have reached a consensus
+  to decline a proposal, the proposal review group moves the proposal to the
+  _Likely Decline_ state.
+
+  A proposal may also be moved to _Likely Decline_ if the proposal review group
+  identifies that no consensus is likely to be reached and that the default of
+  not accepting the proposal is appropriate.
+
+  Similarly to _Likely Accept_, this change should be indicated with a comment
+  on the proposal, and this state marks the final period for comments that
+  might change the recognition of consensus.
+
+* **Accepted**: If a proposal has been in _Likely Accept_ for a week and
+  consensus has not changed, the proposal is moved to the _Accepted_ state.
```

Comment:
I'm not sure if one week is long enough. Sometimes the people who can give the most feedback on a proposal may not be able to take a look for several days. IMO 2 weeks is more reasonable.

Reply:
If there's a specific person that we need input from, then we can wait for them. But by the point a consensus reaches Likely Accept, consensus has already been achieved from the individuals who need to give feedback, as the review committee would've already pinged them.

Also note that changing the transition time from Likely Accept to Accept from one to two weeks also increases the minimum amount of time from 2-3 weeks to 3-4 weeks, which is more than half of a release cycle.

Reply:
To expand on that: "likely accept" means that consensus has already been reached, and relevant people with the most knowledge about a specific topic have already signed off and are a part of the consensus. Likely accept is the final comment period to give external contributors time to raise final concerns and change the current consensus.